src/resources/views/crud/filters/select2_ajax.blade.php

```diff
@@ -76,10 +89,12 @@ class="nav-item dropdown {{ Request::get($filter->name)?'active':'' }}">
 
             	var filterName = $(this).attr('data-filter-name');
                 var filter_key = $(this).attr('data-filter-key');
```

Comment:
```suggestion
                var filterKey = $(this).attr('data-filter-key');
```

Just to keep consistency across this variables.